src/abstractdataframe/abstractdataframe.jl

```diff
@@ -365,39 +369,58 @@ If the column does not allow missing values, `nothing` is returned.
 Consequently, `nmissing = 0` indicates that the column allows
 missing values, but does not currently contain any.
 
+Custom functions perform call `skipmissing` on columns of eltype `Union{T, Missing}`, 
+users should be aware that because of this, they cannot specify special handling
+of missing values in any of the functions given via a `Pair` argument.
+
+
 **Examples**
 
 ```julia
 df = DataFrame(i = 1:10, x = rand(10), y = rand(["a", "b", "c"], 10))
 describe(df)
 describe(df, stats = :all)
-describe(df, stats = [:min, :max])
+describe(df, :min, :max)
+describe(df, :mean, :variance => Statistics.var)
```

Comment:
Need `using Statistics` above. But better use something else like `sum` for simplicity.

Reply:
changed so there is no need for `using Statistics`